MonoGame.Framework/Platform/SDL/SDLGamePlatform.cs

```diff
@@ -233,6 +238,37 @@ private void SdlRunLoop()
                                 break;
                         }
                         break;
+
+                    case Sdl.EventType.DropFile:
+                        if (ev.Drop.WindowId != _view.Id)
+                            break;
+
+                        if (!_view.IsFileDropHandled)
+                            break;
+
+                        unsafe
+                        {
+                            string path = InteropHelpers.Utf8ToString((IntPtr)ev.Drop.File);
+                            Sdl.Drop.SDL_Free(ev.Drop.File);
+                            _dropList.Add(path);
+                        }
+
+                        break;
+
+                    case Sdl.EventType.DropComplete:
+                        if (ev.Drop.WindowId != _view.Id)
+                            break;
+
+                        if (!_view.IsFileDropHandled)
+                            break;
+
+                        if (_dropList.Count > 0)
+                        {
+                            _view.OnFileDrop(new FileDropEventArgs(_dropList.ToArray()));
```

Comment:
OnFileDrop will protect you against null check for the event, just pointing it out so you don't add a null check for the event.